tests/admin_changelist/test_date_hierarchy.py

```diff
@@ -11,9 +12,14 @@
 class DateHierarchyTests(TestCase):
     factory = RequestFactory()
 
+    def setUp(self):
+        # We make requests as superuser (so that he has change permissions on all models)
```

Comment:
Please rephrase the comment to avoid personal pronouns.

Reply:
About the save and continue: I'll change it so that it only appears when the user can change, and to reword to "Save and view" if the user can only view (good catch !)